test/standalone.zig

```diff
@@ -230,6 +230,10 @@ pub const build_cases = [_]BuildCase{
         .build_root = "test/standalone/cmakedefine",
         .import = @import("standalone/cmakedefine/build.zig"),
     },
+    .{
+        .build_root = "test/standalone/issue_15920",
+        .import = @import("standalone/issue_15920/build.zig"),
```

Comment:
Please avoid creating tests named after GitHub issue numbers. Prefer to use descriptive name that explains what is being tested.